ykfr/src/lib.rs

```diff
@@ -6,12 +8,32 @@ use std::{
     env,
     ffi::{c_void, CStr},
     fs, ptr, slice,
+    sync::LazyLock,
 };
-use yksmp::{Location as SMLocation, StackMapParser};
+use yksmp::{Location as SMLocation, SMEntry, StackMapParser};
 
 mod llvmbridge;
 use llvmbridge::{get_aot_original, Module, Type, Value};
 
+pub static AOT_STACKMAPS: LazyLock<Vec<SMEntry>> = LazyLock::new(|| {
+    // Load the stackmap from the binary to parse in the stackmaps.
+    // FIXME: Don't use current_exe.
+    let pathb = env::current_exe().unwrap();
+    let file = fs::File::open(&pathb.as_path()).unwrap();
```

Comment:
There are several places in the system where the stackmap section is being loaded from disk. At some point we should have ykllvm mark the section loadable and fish that data out of the virtual address space.

Can you add a FIXME for this please?

Reply:
I'm not aware of other places where we parse the AOT stackmap. The only other place where `StackmapParser` is used is to get the stackmaps of a trace.

Reply:
Admittedly I only did a quick grep for the string `".llvm_stackmaps"`, which appears in many places.